### ISSUE TITLE
UI Command and subcommands

### ISSUE THREAD

**Issue by InbalZilberman** [cyberark/conjur-api-python3#274]
## **Feature Overview & Customer Need**

**As a**  Conjur user 
**I would like to** use the Conjur cli 
**In order to** open the Conjur UI central page

**As a**  Conjur user 
**I would like to** use the Conjur cli 
**In order to** open the Conjur UI specific pages according to the context we are in.

A new command will be called ui
`conjur ui`
will open the dashboard of the Conjur UI (or login page if needed)

Also for the following commands a new subcommand will be provided called --ui: policy, host, user, variable
This subcommand will open the relevant page of the UI
for example
`conjur policy --ui `
will open the screen 
@benoaviram  to provide a picture

## Help for command
Use the ui command to Conjur Enterprise UI in the local default browser.
For Conjur deployment without a UI is will provide a proper error
**Usage**
 conjur [global options] ui
**Global options**
Option | Description
-d, --debug | Enable debugging output
**Options**
-h, --help | Display the help screen
Example
conjur ui -  The following command opens the  Conjur Enterprise UI in the local default browser (edited) 

## Failure scenario
If we are working with OSS  - @mbenita-Cyberark  to fill in
If there is no browser in the machine -  @mbenita-Cyberark  to fill in

## Quality

Make sure we have test coverage of the ui command and subcommands.  Create test plan and execute accordingly.

## Process logic and Demo

Conjur user with a machine that the Python CLI already been installed and conjur init & login run against a Conjur/ Conjur Enterprise machine 
The user wants to open the dashboard he runs
`conjur ui`
and a browser is opened with the dashboard of Conjur UI.

The user has loaded policies, set variables and so on
now he would like to view his variables in his Conjur. He runs:
`conjur variable --ui`

Demo the flow above and the failure scenario
## User messages

All user messages regarding resource actions should be reviewed

Especially error messages 

if an argument is missing we need to return the help of the command 

## **Documentation**

Please provide enhance documentation in online help and readme

## DOD

- [ ] Implement the UI command and subcommands in Conjur Python CLI in all 3 platforms
- [ ] Demo the flow described above
- [ ] Automatic integration tests written according to a test plan and passed successfully
- [ ] Fill in the user messages - link to a page with all user messages for CLI saved and reviewed. 
- [ ] Fill in the logs and place here a link to the logs files - link to a page with all logs for CLI saved and reviewed. 
- [ ] Logs were reviewed by TW and PO
- [ ] Are there new Audits to this feature? If so please document them and provide here a link (N/A)

**Comment by shulifink** [comment]
I have to say that the UI looks just like the Conjur OSS site (has the identical look and feel, and says Cyberark conjur" at the top, so it's quite misleading that it's for enterprise only.

OSS Site:
![image.png](https://images.zenhubusercontent.com/5ea1737ff37aec9c9b1af055/cc8cbbae-1bdd-4437-978f-090017ef0f64)

Enterprise UI:
![image.png](https://images.zenhubusercontent.com/5ea1737ff37aec9c9b1af055/135deeff-3397-415d-b4e1-ef8c33b441cf)

OSS Docs:
![image.png](https://images.zenhubusercontent.com/5ea1737ff37aec9c9b1af055/79f2ad1f-2e6e-418e-8cb6-0bada16e4a5c)

Is the UI going to be "rebranded" for enterprise?

**Comment by InbalZilberman** [comment]
@shulifink good question - I believe need to ask PM about this

**Comment by rafis3** [comment]
@InbalZilberman, @sigalsax I think it would be nice if the UI option would not be specified in the list of available commands under `--help` if working with Conjur OSS. The CLI can figure out the type of instance in the `conjur init` command and keep this information for the execution of the other commands.

**Comment by sigalsax** [comment]
@rafis3 If I am understanding correctly you are suggesting that the UI option only appear if Conjur enterprise is detected?

**Comment by rafis3** [comment]
Yes. So that a user will get an accurate list of supported commands.

**Comment by InbalZilberman** [comment]
@rafis3 Nice!, today we are indicating the Enterprise by the existence of /info. Ill Add it to the feature definition.
@mbenita-Cyberark FYI
